api_doc/ecoChange.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @apiDefine s_ecoChange Search > Forest
+ * Information related to forest  in a given geofence.
```

Comment:
there is an extra space between "forest" and "in"

Reply:
Done